src/devices/device_models/AC_branches.jl

```diff
@@ -120,14 +108,69 @@ function branch_rate_constraint(ps_m::CanonicalModel,
 
     rating_constraint(ps_m,
                         range_data,
-                        Symbol("rate_limit_fwd_$(B)"),
-                        (Symbol("Pbr_fwd_$(B)"), Symbol("Qbr_fwd_$(B)")))
+                        Symbol("RateLimitFT_$(B)"),
+                        (Symbol("FpFT_$(B)"), Symbol("FqFT_$(B)")))
 
     rating_constraint(ps_m,
                         range_data,
-                        Symbol("rate_limit_bwd_$(B)"),
-                        (Symbol("Pbr_bwd_$(B)"), Symbol("Qbr_bwd_$(B)")))
+                        Symbol("RateLimitTF_$(B)"),
+                        (Symbol("FpTF_$(B)"), Symbol("FqTF_$(B)")))
 
     return
 
 end
+
+function branch_rate_constraint(ps_m::CanonicalModel,
+    devices::PSY.FlattenIteratorWrapper{B},
+    device_formulation::Union{Type{StaticLineUnbounded}, Type{StaticTransformerUnbounded}},
+    system_formulation::Type{S}) where {B <: PSY.ACBranch,
+                                        S <: PM.AbstractPowerFormulation}
+
+    # This code is intended to do nothing
+
+    return
+
+end
+
+
+function branch_flow_constraint(ps_m::CanonicalModel,
+                                devices::PSY.FlattenIteratorWrapper{PSY.MonitoredLine}, # TODO: Do we need an AbstractMonitoredLine?
+                                device_formulation::Type{FlowMonitoredLine},
+                                system_formulation::Union{Type{PM.DCPlosslessForm}, Type{StandardPTDFForm}})
+
+
+    flow_range_data = [(PSY.get_name(h), PSY.get_flowlimits(h)) for h in devices]
+
+    var_name = Symbol("Fp_$(B)")
+
+    device_range(ps_m,
+                range_data,
+                Symbol("FlowLimit_$(B)"),
+                var_name)
+
+    return
+
+end
+
+function branch_flow_constraint(ps_m::CanonicalModel,
+                                devices::PSY.FlattenIteratorWrapper{PSY.MonitoredLine},
+                                device_formulation::Type{FlowMonitoredLine},
+                                system_formulation::Type{S}) where {S <: PM.AbstractPowerFormulation}
+
+    FTflow_range_data = [(PSY.get_name(h), PSY.get_flowlimits(h)) for h in devices]
```

Comment:
Why are these different?

Reply:
if `Line1` has a `flowlimits(min=-100,max=200)` we need to keep track of the directionality of the values.